docs/spec/index.html

```diff
@@ -1,740 +1,952 @@
+      document.
+    </p>
+  </section>
+
+  <section data-include="sections/use-case-requirements.html"
+    data-include-replace="true"></section>
+
+  <section data-include="sections/authorization.html"
+    data-include-replace="true"></section>
+
+  <section data-include="sections/api-spec.html" data-include-replace="true">
+  </section>
+
+  <section class="informative">
+    <h3>Workflows</h3>
+    <p>
+      <em>Workflows</em> are a critical concept in this specification that
+      enable presenters to correlate individual
+      presentations. A workflow <em>definition</em> can be used to indicate that
+      presentations relate to the same
+      <em>type</em>
+      of workflow.
+    </p>
+    <p>
+      Workflow <em>instances</em> indicate that separate individual
+      presentations are part of the same
+      <em>instance</em> of a specified workflow <em>type</em>.
+    </p>
 
+    <section class="informative">
+      <h3>Workflows for US Customs and Border Protection (US CBP)</h3>
+      <img src="resources/workflow-def-ex.png" alt="workflow-def-ex"
+        width="75%" />
       <p>
-        To contribue to this vocabulary or reference technical details related
-        to the project, please reference the primary README located on
-        <a href="https://github.com/w3c-ccg/traceability-interop/">github</a>
+        In the above example, two separate 7501 Entry Summaries are
+        presented to US CBP at different points in time, and related to
+        different imports. By specifying the same workflow
+        <em>definition</em>, it is made clear to US CBP that both presentations
+        relate to the same <em>type</em> of
+        import workflow.
       </p>
-
+      <img src="resources/workflow-inst-ex.png" alt="workflow-inst-ex"
+        width="75%" />
       <p>
-        Please
-        <a href="https://github.com/w3c-ccg/traceability-interop/issues/"
-        >open an issue</a
-        >, if you wish to collaborate on this specification.
+        The majority of data elements in a "10+2" Importer Security
+        Filing must be presented to CBP no later than 24 hours before
+        vessel loading, and the remaining data elements no later than 24
+        hours before arrival. Specifying the same workflow
+        <em>instance</em> indicates to US CBP that the two presentations are
+        part of the same import workflow.
       </p>
-
       <p>
-        You may also reach out via the mailing list:
-        <a href="mailto:public-credentials@w3.org">public-credentials@w3.org</a>
-        (<a href="mailto:public-credentials-request@w3.org?subject=subscribe"
-        >subscribe</a
-        >,
-        <a href="https://lists.w3.org/Archives/Public/public-credentials/"
-        >archives</a
-        >)
+        A workflow definition indended for use with US CBP imports
+        <em>MUST</em>
+        have a corresponding plain text
+        <em>workflow manifest document</em> that may be uniquely
```

Comment:
new requirement, embedded in a massive change-set... please include an issue to track discussion for this.

Reply:
@brownoxford @mprorock please don't resolve comments like this... There should be an issue added to to the spec.

Reply:
this requirement has been in for months.... this PR just forced people to read spec top to bottom

Reply:
issue #505 opened on the topic

Reply:
@OR13 I have added a link to #495, which I believe covers this and your next comment.